### DISCUSSION TITLE
How to set postgres server version?

### DISCUSSION THREAD

**Discussion by a137872798** [GreptimeTeam/greptimedb#2681] [Q&A]
I start greptimeDB server and test jdbc.When i user prepareStatement, the response show Unsupported Server Version: 0.4.2. I find  
the var associated with env CARGO_PKG_VERSION. I restart server with docker -e CARGO_PKG_VERSION=14.9 and found the env in container.But when I request again, the error still exist.

**Comment by sunng87** [comment]
hi @a137872798 thanks for reporting the issue. May I know your JDBC driver version? Does it only happen when using prepared statement? 

I think we will be hard-coding a fixed version that satisfies most client drivers.

**Comment by a137872798** [comment]
My java postgres driver version is 
![image](https://github.com/GreptimeTeam/greptimedb/assets/35627706/2f200238-e777-49ea-afae-0b7ae632870a)
and my postgres client version is 14.9.
By the way, are there any emergency measures to solve the problem.

**Comment by a137872798** [comment]
I had solve, thank you
